.github/workflows/build_tag_no_latest.yaml

```diff
@@ -0,0 +1,22 @@
+name: Build and push a release without :latest
+
+on:
+  push:
+    tags:
+      - 'v[0-9]*.[0-9]*.[0-9]*beta*'
+      - 'v[0-9]*.[0-9]*.[0-9]*alpha*'
+      - 'v[0-9]*.[0-9]*.[0-9]*rc*'
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Should we pick a specific version to rule out potential `latest` issues?

Reply:
I don't think so at the moment, this is only for doing Docker builds, so all the actual building is done in the go Docker image anyway.